tests/test_packages/test_connections/test_p2p_libp2p/test_public_dht.py

```diff
@@ -105,174 +85,72 @@ def delegate_uris_public_keys(request):
 class TestLibp2pConnectionPublicDHTRelay(BaseP2PLibp2pTest):
     """Test that public DHT's relay service is working properly"""
 
-    @pytest.mark.parametrize(
-        "maddrs", [PUBLIC_DHT_MADDRS, PUBLIC_STAGING_DHT_MADDRS], indirect=True
-    )
+    @pytest.mark.parametrize("maddrs", [PUBLIC_DHT_MADDRS], indirect=True)
```

Comment:
we dont have public and staging. I left the test parameterized so we can add and differentiate between such if / when needed later